server/sonar-web/src/main/js/components/issue/actions.js

```diff
@@ -48,6 +48,7 @@ export const updateIssue = (
       }
     },
     error => {
+      console.log(`testlog`, error); // eslint-disable-line
```

Comment:
:man_facepalming: I need to find a way to better check this before my commits... Maybe a hook or something...